prestashop1.6/views/templates/admin/tawkto/helpers/view/view.tpl

```diff
@@ -15,123 +15,81 @@
  * @copyright Copyright (c) 2014-2021 tawk.to
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  *}
-
 {include file="toolbar.tpl" toolbar_btn=$toolbar_btn toolbar_scroll=$toolbar_scroll title=$title}
 
-{if !$same_user}
-<div id="widget_already_set" style="width: 50%; float: left; color: #3c763d; border-color: #d6e9c6; font-weight: bold;" class="alert alert-warning">Widget set by other user</div>
-{/if}
-
-<iframe
```

Comment:
This iframe was moved in [line 44](https://github.com/tawk/tawk-prestashop/pull/17/files#diff-2f067937ab3c16db22129b4ff143c0d6dda6b33cb00e14a8a60ee6e4851c34d2R44)